content/en/blog/2020/large-scale-security-policy-performance-tests/index.md

```diff
@@ -0,0 +1,148 @@
+
+{{< tabset category-name="platform" >}}
+
+{{< tab name="MTLS vs plainText" category-value="one" >}}
+{{< image width="90%" ratio="45.34%"
+    link="./mtls_plaintext.svg"
+    alt="MTLS vs plaintext"
+    caption=""
+    >}}
+The difference between MTLS mode STRICT vs that of plaintext is very small in lower loads and as the number of qps and conn increase the latency of requests with MTLS STRICT rise higher. The additional latency increased in larger loads is minimal compared to that of the increase from having no sidecars vs having sidecars in the plaintext.
+{{< /tab >}}
+
+{{< tab name="AuthZ principals" category-value="two" >}}
+{{< image width="90%" ratio="45.34%"
+    link="./AuthZ_var_principals.svg"
+    alt="Authorization policy variable number of principals"
+    caption=""
+    >}}
+
+For Authorization policies with 10 vs 1000 principal rules, there is a larger increase of latency of having 10 principal rules compared to having no policies than there is with the increase of latency of having 1000 principals compared to having 10 principals.
+{{< /tab >}}
+
+{{< tab name="AuthZ requestPrincipals" category-value="three" >}}
+{{< image width="90%" ratio="45.34%"
+    link="./AuthZ_var_requestPrincipals.svg"
+    alt="Authorization policy with variable principals"
+    caption=""
+    >}}
+    For Authorization policies with a variable number of `requestPrincipal` rules, the difference of latency of having 10 `requestPrincipal` rules compared to no policies is nearly the same as the increase of having 1000 `requestPrincipal` rules compared to having 10 `requestPrincipal` rules.
+{{< /tab >}}
+
+{{< tab name="paths vs sourceIP" category-value="four" >}}
+To compare the latency of having a variable number of path vs `sourceIP` the data is broken into 3 graphs.
+The first showing the data of having a variable number of `sourceIP` rules in a single Authorization policy as well as the baseline.
+{{< image width="90%" ratio="45.34%"
+    link="./AuthZ_var_sourceIP.svg"
+    alt="Authorization policy with variable `sourceIP` rules"
+    caption=""
+    >}}
+The second graph shows the data of having a variable number of path rules in a single Authorization policy as well as the baseline.
+{{< image width="90%" ratio="45.34%"
+    link="./AuthZ_var_paths.svg"
+    alt="Authorization policy with variable number of paths"
+    caption=""
+    >}}
+And finally the last graph shows the data of having a variable number of `sourceIP` rules vs path rules without the baseline.
+{{< image width="90%" ratio="45.34%"
+    link="./AuthZ_paths_vs_sourceIP.svg"
+    alt="Authorization policy with both paths and `sourceIP`"
+    caption=""
+    >}}
+Having `sourceIP` rules increases the latency of request in a minimal amount compared to that of path rules.
+{{< /tab >}}
+
+{{< tab name="JWTRule" category-value="five" >}}
```

Comment:
can we name this as "RequeatAuthN JWT Issuer", variable number of `JWTRules` really mean many different issuers, so call out issuer is more familiar and meaningful to (new) users.

Reply:
Makes sense, thanks